swan-lake/integration-tools/health-tool.md

```diff
@@ -1,90 +1,150 @@
 ---
 layout: ballerina-health-support-left-nav-pages-swanlake
 title: Health tool (FHIR/HL7)
-description: The health tool generates API templates and library packages for developing healthcare integrations.
-keywords: ballerina, programming language, health, contract
+description: The health tool generates Ballerina library packages and Ballerina templates for FHIR APIs for developing healthcare integrations.
+keywords: ballerina, programming language, healthcare, contract
 permalink: /learn/health-tool/
 active: health-tool
-intro: The health tool generates API templates and library packages for developing healthcare integrations.
+intro: The health tool generates Ballerina library packages and Ballerina templates for FHIR APIs for developing healthcare integrations.
 ---
 
-They are primarily focused on the [Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard and are developed using Ballerina.
+These artifacts generated by the tool are primarily focused on the [Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard and are developed using Ballerina.
 
->**Info:** It is supported from Ballerina Swan Lake version 2201.7.0 onwards.
+>**Info:** It is supported from Ballerina Swan Lake version 2201.8.1 onwards.
```

Comment:
didn't we support health tool from update 7 onwards? Reason to change?

Reply:
Our latest tool only works with bal 8 since the bal tool framework is updated to work with bal 8 and java 17. So the tool generates artifacts that is compatible with bal 8. Also the new FHIR API 'template' mode is only added with the latest tool version which will not work with bal 7.

Reply:
Ack.